tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
+  await page.goto('./');
+  await login(page, context);
+
+  // Creating and navigating to new post
+  await page.goto('./wp-admin/post-new.php');
+
+  // On dev instance, need to close modal so test can continue
+  const closeButton = page.locator('.components-modal__header button');
+  if (await closeButton.isVisible()) {
+    await closeButton.click();
+  }
+
+  await page.locator('.editor-post-title__input').click();
+  await page.locator('h1.editor-post-title').fill('Test Post');
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('This is a test Post.');
+  await page.keyboard.press('Enter');
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/youtube');
+  await page.keyboard.press('Enter');
+  await page.locator('[aria-label="YouTube URL"]').fill('https://youtu.be/3gPvDDHU41E');
+  await page.getByRole('button', {name: 'Embed'}).click();
+  await page.getByRole('button', {name: 'Publish', exact: true}).click();
+  await page.getByRole('region', {name: 'Editor publish'}).getByRole('button', {name: 'Publish', exact: true}).click();
+  await page.getByRole('link', {name: 'View Post', exact: true}).first().click();
```

Comment:
@mleray, this is one of the reasons I use the names of the elements to locate them. In this case to minimise the amount of code written and make sure the tests run both on _local_ and _dev-instances_, I use this `selector` to catch any instances of the `link` _"View Post"_, on _local_ it appears once but on the _dev-instance_ it appears twice, so just the first instance is clicked. Also I realised using these `selectors` with names, seem to work faster when running tests.

Reply:
I guess it's okay to keep as is then. Just something to keep in mind for certain test instances where it might cause issues due to a different language!